wrappers/swift/PrivacyGuard.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Wrapper to `ODWPrivacyGuard` representing Privacy Guard Hook.
+public class PrivacyGuard {
```

Comment:
Do we expect subclasses of this class? If not, we should mark this as `final` to prevent unexpected behaviors if someone chooses to subclass.